### DISCUSSION TITLE
Load pages for FPL standings

### DISCUSSION THREAD

**Discussion by DucLMN** [hoothin/UserScripts#377] [Q&A]
Hi hoothin, please help me to check if Pagetual can load this standing or not ? 
Link: https://fantasy.premierleague.com/leagues/5198/standings/c?phase=1&page_new_entries=1&page_standings=2

Thanks you ^^

**Comment by hoothin** [comment]
<pre name="pagetual">
{
    "name": "Fantasy Premier League",
    "url": "^https?://fantasy\\.premierleague\\.com/leagues/",
    "pageElement": "#root table",
    "waitElement": "#root table",
    "action": 1
}
</pre>

https://pagetual.hoothin.com/en/rules/waitElement.html
